pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,43 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            let path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{path}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
+            } else if (target.type === "ssh") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through SSH to $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{`${target.server}:${path}`}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
+            } else if (target.type === "nfs") {
+                path = path[0] === '/' ? path : '/' + path;
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through NFS at $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{`${target.server}:${target.export}${path}`}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
```

Comment:
`verifyMessage` is constructed the same way three times with the only big difference being the translated text and the destination. It seems more logical to construct the `verifyMessage` once and simply assign the variables in if/else if/else if statements.

Reply:
First message is too different from the other 2 messages, since it for local filesystem it says `stored in $0 as $1`, and for nfs/ssh it says `copied through NFS/SSH to $0 as $1`. Making the template would mean breaking the sentence apart, which would complicate translations (and probably using proper grammar impossible) in some languages. 

But I guess I can use the same template at least for NFS/SSH